src/CTA.WebForms2Blazor/ClassConverters/ControlCodeBehindClassConverter.cs

```diff
@@ -1,24 +1,52 @@
-﻿using System;
-using System.Threading.Tasks;
+﻿using System.Threading.Tasks;
 using CTA.WebForms2Blazor.FileInformationModel;
+using CTA.WebForms2Blazor.Helpers;
+using CTA.WebForms2Blazor.Extensions;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System.Linq;
+using System.Text;
+using System.IO;
 
 namespace CTA.WebForms2Blazor.ClassConverters
 {
     public class ControlCodeBehindClassConverter : ClassConverter
     {
         public ControlCodeBehindClassConverter(
             string relativePath,
+            string sourceProjectPath,
             SemanticModel sourceFileSemanticModel,
+            TypeDeclarationSyntax originalDeclarationSyntax,
             INamedTypeSymbol originalClassSymbol)
-            : base(relativePath, sourceFileSemanticModel, originalClassSymbol)
+            : base(relativePath, sourceProjectPath, sourceFileSemanticModel, originalDeclarationSyntax, originalClassSymbol)
         {
-
+            // TODO: Register with the necessary services
         }
 
         public override async Task<FileInformation> MigrateClassAsync()
         {
-            throw new NotImplementedException();
+            // NOTE: For now we make no code modifications, just to be
+            // ready for the demo and produces files
+            var requiredNamespaces = _sourceFileSemanticModel.GetNamespacesReferencedByType(_originalDeclarationSyntax);
+            var usingStatements = CodeSyntaxHelper.BuildUsingStatements(requiredNamespaces.Select(namespaceSymbol => namespaceSymbol.Name));
+            var namespaceNode = CodeSyntaxHelper.BuildNamespace(_originalClassSymbol.ContainingNamespace.Name, _originalDeclarationSyntax);
+            var fileText = CodeSyntaxHelper.GetFileSyntaxAsString(namespaceNode, usingStatements);
```

Comment:
Seems like these 4 lines are repeated in a few of the converters. Would it make sense to define a function in the parent class that returns this information in a new object called `SourecFileComponents` or something similar?